src/3d/framegraph/qgspostprocessingentity.cpp

```diff
@@ -46,16 +46,19 @@ typedef Qt3DCore::QGeometry Qt3DQGeometry;
 #include "qgsframegraph.h"
 #include "qgsshadowrenderview.h"
 #include "qgsforwardrenderview.h"
+#include "qgsambientocclusionrenderview.h"
 
 QgsPostprocessingEntity::QgsPostprocessingEntity( QgsFrameGraph *frameGraph, Qt3DRender::QLayer *layer, QNode *parent )
   : QgsRenderPassQuad( layer, parent )
 {
   QgsShadowRenderView &shadowRenderView = frameGraph->shadowRenderView();
   QgsForwardRenderView &forwardRenderView = frameGraph->forwardRenderView();
+  QgsAmbientOcclusionRenderView *aoRenderView = dynamic_cast<QgsAmbientOcclusionRenderView *>( frameGraph->renderView( QgsFrameGraph::AO_RENDERVIEW ) );
```

Comment:
please add ambientOcclusionRenderView() to frame graph public API to hide this (like above with forward/shadow render views)

Reply:
done